public/js/md-editor.js

```diff
@@ -36,3 +36,7 @@ const easyMDE = new EasyMDE({
     }
   }]
 })
+
+if (typeof description !== 'undefined') {
```

Comment:
I personally like to use `myVar != null`, because it checks for null as well as undefined.